helm/kfp-operator/values.yaml

```diff
@@ -51,13 +51,22 @@ manager:
   webhookServicePort: 9443
   runcompletionWebhook:
     servicePort: 8082
-    endpoints: []
+    endpoints:
```

Comment:
As config comment think this should be [] for default install, no event feedback by default